sickbeard/showUpdater.py

```diff
@@ -92,43 +92,43 @@ def run(self, force=False):  # pylint: disable=unused-argument, too-many-locals,
         except SyntaxError:
             pass
 
-        logger.log(u"Doing full update on all shows")
+        logger.log(u'Doing full update on all shows')
```

Comment:
I'm changing it to a logger.info now.. just caught me out as that worked
with the previous logger and I presumed it would with the new one.

On Sat, Jul 23, 2016 at 11:35 AM, Labrys of Knossos <
notifications@github.com> wrote:

> In sickbeard/showUpdater.py
> <https://github.com/pymedusa/SickRage/pull/778#discussion_r71976278>:
>
> > @@ -92,43 +92,43 @@ def run(self, force=False):  # pylint: disable=unused-argument, too-many-locals,
> >          except SyntaxError:
> >              pass
> >
> > -        logger.log(u"Doing full update on all shows")
> > +        logger.log(u'Doing full update on all shows')
>
> @Thraxis <https://github.com/Thraxis> I think its because you are doing a
> logger.log instead of for example logger.info and you arent passing a
> logging level.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/pymedusa/SickRage/pull/778/files/ab1f62e989c870556e59de5e5ddb4f3373d00c12#r71976278>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABFFgd9WMjy-C4efaaA-nLvBIcxP_Y8Oks5qYl8EgaJpZM4JOH70>
> .
>